util/Setup/EnvironmentFileBuilder.cs

```diff
@@ -170,9 +170,6 @@ private void Build()
             }
 
             Helpers.Exec("chmod 600 /bitwarden/env/mssql.override.env");
-
-            // Empty uid env file. Only used on Linux hosts.
-            using(var sw = File.CreateText("/bitwarden/env/uid.env")) { }
```

Comment:
This has to stay for windows hosts since the docker file references a uid.env on all hosts

Reply:
Ah yes, I did not think about Win hosts.
I added a test to create the file only if it does not exist.